### PR TITLE
upgrade llvm 9.0 to resolve xcode11 build's issue

### PR DESCRIPTION
**Pull request by skywancen**

### CODE REVIEW

**Comment by ryuichis** [comment]
@skywancen you may want to take a look at this commit: https://github.com/ryuichis/oclint/commit/01477dff6226bc965f2e331bcc7afad65c263bff

File: oclint-scripts/oclintscripts/version.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def dev_version():
         return "src"
 
 def oclint_version():
-    return "0.14"
+    return "0.15.0"
```
Comment (ryuichis) [inline oclint-scripts/oclintscripts/version.py]: I am thinking to pump it to `0.15` without the trailing `.0`.

File: oclint-scripts/oclintscripts/path.py [hunk 2]
```diff
@@ -73,8 +73,8 @@ class url:
     json_compilation_database = 'https://github.com/oclint/oclint-json-compilation-database.git'
     xcodebuild = 'https://github.com/oclint/oclint-xcodebuild.git'
 
-    clang_prebuilt_binary_for_macos = 'http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-apple-darwin.tar.xz'
-    clang_prebuilt_binary_for_ubuntu_lts_16 = 'http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz'
+    clang_prebuilt_binary_for_macos = 'http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-darwin-apple.tar.xz'
+    clang_prebuilt_binary_for_ubuntu = 'http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-14.04.tar.xz'
```
Comment (ryuichis) [inline oclint-scripts/oclintscripts/path.py]: I second your renaming of this variable, but if you do a global search, you will find that two other places (if I remember correctly) use this variable, where you also need to update.

Comment (skywancen) [reply oclint-scripts/oclintscripts/path.py]: Oh！It's my fault，I just want to change the value inside，not the variable name.
I'll change it right away。

**Pull request merged**